docs/source/releasenotes.rst

```diff
@@ -3,8 +3,29 @@ Release Notes
 
 This page lists the changes made in each point version of gpkit.
 
+Version 0.5.0
+=============
+ * No longer recommend the use of linked variables and subinplace
+ * Switched default solver to MOSEK
+ * Added Linked Variable diagram (PR #915)
+ * Dropped operator overloading with pint (PR #938)
```

Comment:
More like "Changed how overloaded operators interact with pint"? (we didn't drop our operator overloading to form constraints, which is how I might read the above)